python/lsst/ts/vent/controller/controller.py

```diff
@@ -246,6 +307,10 @@ def vent_open(self, vent_number: int) -> None:
 
         Raises
         ------
+        ModuleNotFoundError
```

Comment:
So, you are not really raising this exception here. But we should talk abut this.